test/unit/policy_test.rb

```diff
@@ -194,37 +194,39 @@ class PolicyTest < ActiveSupport::TestCase
     end
   end
 
-  test 'policy not destroyed if still referenced by assets' do
-    policy = FactoryBot.create(:public_policy)
-    sample_type = FactoryBot.create(:strain_sample_type)
-    data_file = FactoryBot.create(:strain_sample_data_file, policy: policy)
-    samples = data_file.extract_samples(sample_type, true).select(&:persisted?)
-    sample = samples.first
-
-    assert_equal sample.policy, data_file.policy
-
-    assert_no_difference('Policy.count') do
-      disable_authorization_checks { data_file.destroy }
-    end
-
-    assert_not_nil sample.reload.policy
-    assert_not_nil Policy.find_by_id(policy.id)
-  end
-
-  test 'policy destroyed when no longer referenced' do
-    policy = FactoryBot.create(:public_policy)
-    sample_type = FactoryBot.create(:strain_sample_type)
-    data_file = FactoryBot.create(:strain_sample_data_file, policy: policy)
-    samples = data_file.extract_samples(sample_type, true).select(&:persisted?)
-
-    disable_authorization_checks { data_file.destroy }
-
-    assert_difference('Policy.count', -1) do
-      disable_authorization_checks { samples.each(&:destroy) }
-    end
-
-    assert_nil Policy.find_by_id(policy.id)
-  end
+  # Samples no longer reference the policy, they make a copy of it.
```

Comment:
delete these tests, rather than leave commented out

Reply:
Instead of deleting them I replaced samples with two sops forced to reference same policy in [7bac631](https://github.com/seek4science/seek/pull/1599/commits/7bac631a6b0394eff078f81fc3e1c13a1611a289).